taskt/Core/Automation/Commands/DataTable Commands/AddDataRowCommand.cs

```diff
@@ -1,148 +1,147 @@
-            {
-                //reformat and attempt
-                var reformattedVariable = v_DataTableName.Replace(sendingInstance.EngineSettings.VariableStartMarker, "").Replace(sendingInstance.EngineSettings.VariableEndMarker, "");
-                requiredVariable = sendingInstance.VariableList.Where(var => var.VariableName == reformattedVariable).FirstOrDefault();
-            }
 
-            return requiredVariable;
-        }
         public override List<Control> Render(frmCommandEditor editor)
         {
             base.Render(editor);
 
-            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_DataTableName", this, editor));
-
+            RenderedControls.AddRange(CommandControls.CreateDefaultInputGroupFor("v_DataTable", this, editor));
+            RenderedControls.AddRange(CommandControls.CreateDataGridViewGroupFor("v_DataRowDataTable", this, editor));
 
-            RenderedControls.AddRange(CommandControls.CreateDataGridViewGroupFor("v_AddDataDataTable", this, editor));
-
-            taskt.UI.CustomControls.CommandItemControl loadSchemaControl = new taskt.UI.CustomControls.CommandItemControl();
+            CommandItemControl loadSchemaControl = new CommandItemControl();
             loadSchemaControl.ForeColor = System.Drawing.Color.White;
-            loadSchemaControl.CommandDisplay = "Load Column Names From Existing Table";
+            loadSchemaControl.CommandDisplay = "Load Column Names From Existing DataTable";
             loadSchemaControl.Click += LoadSchemaControl_Click;
             RenderedControls.Add(loadSchemaControl);
 
-            DataTableCreationCommands = editor.ConfiguredCommands.Where(f => f is CreateDataTableCommand).Select(f => (CreateDataTableCommand)f).ToList();
+            _dataTableCreationCommands = editor.ConfiguredCommands.Where(f => f is CreateDataTableCommand)
+                                                                 .Select(f => (CreateDataTableCommand)f)
+                                                                 .ToList();
 
+            return RenderedControls;
+        }
 
+        public override string GetDisplayValue()
+        {
+            return base.GetDisplayValue() + $" [Add {v_DataRowDataTable.Rows.Count} Field(s) to '{v_DataTable}']";
+        }
 
+        private ScriptVariable LookupVariable(AutomationEngineInstance sendingInstance)
+        {
+            //search for the variable
+            var requiredVariable = sendingInstance.VariableList.Where(var => var.VariableName == v_DataTable).FirstOrDefault();
 
-            return RenderedControls;
+            //if variable was not found but it starts with variable naming pattern
+            if ((requiredVariable == null) && (v_DataTable.StartsWith(sendingInstance.EngineSettings.VariableStartMarker)) && (v_DataTable.EndsWith(sendingInstance.EngineSettings.VariableEndMarker)))
```

Comment:
Please split this statement by **&&** operator for readability purpose.
Do same for the line 115 below. (Split by .Replace)

Reply:
Done